include/dlr.h

```diff
@@ -47,8 +60,34 @@ typedef void* DLRModelHandle;
  success, -1 for error. Call DLRGetLastError() to get the error message.
  */
 DLR_DLL
-int CreateDLRModel(DLRModelHandle* handle, const char* model_path, int dev_type,
-                   int dev_id);
+int CreateDLRModel(DLRModelHandle* handle, const char* model_path, int dev_type, int dev_id);
+
+/*!
+ * \brief Creates a TVM DLR model.
```

Comment:
Can you expand a bit more in the docstring on the difference between this and a regular DLR model? Maybe clarify this is for advanced users.

Can we also group all the TVM specific APIs together, perhaps at the end of the file? Most regular users probably won't uses these and it might be confusing.

Reply:
done